src/integrationTest/groovy/wooga/gradle/build/unity/ios/tasks/KeychainTaskSpec.groovy

```diff
@@ -97,4 +97,18 @@ class KeychainTaskSpec extends IntegrationSpec {
         then:
         result.wasUpToDate('buildKeychain')
     }
+
+    def "fails with security stderr printed to error log"() {
+        given: "wrong certificatePassphrase"
+        buildFile << """
+            iosBuild.certificatePassphrase = "randomPassphrase"
+        """.stripIndent()
+
+        when:
+        def result = runTasksWithFailure('buildKeychain')
+
+        then:
+        outputContains(result, "security: SecKeychainItemImport: MAC verification failed during PKCS12 import (wrong password?)")
```

Comment:
If Apple guys change the punctuation or phrasing this test will stop working, right? Maybe checking for only "verification failed" would be more future-proof? Not sure.

Reply:
Yes. I wanted to add a real world test. I'll think about a better way of testing this.